libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts

```diff
@@ -505,9 +505,11 @@ export const NewPrimarySchoolForm: Form = buildForm({
       ],
     }),
     buildFormConclusionSection({
-      alertTitle: newPrimarySchoolMessages.conclusion.alertTitle,
-      expandableHeader: newPrimarySchoolMessages.conclusion.nextStepsLabel,
-      expandableDescription: newPrimarySchoolMessages.conclusion.accordionText,
+      alertMessage: newPrimarySchoolMessages.conclusion.alertMessage,
+      expandableHeader: newPrimarySchoolMessages.conclusion.expandableHeader,
```

Comment:
I think you can remove this line, this is the same text that is default in the `buildFormConclusionSection`. Remember to also remove from `messages.ts`
```suggestion
```

Reply:
Removed